extensions/arc/src/ui/components/dashboardPage.ts

```diff
@@ -28,7 +28,11 @@ export abstract class DashboardPage extends InitializingComponent {
 			icon: this.icon,
 			// Get toolbar first since things in the container might depend on these being created
 			toolbar: this.toolbarContainer,
-			content: this.container
+			content: this.container,
+			dispose: () => {
```

Comment:
Make sure you're tracking which extensions will need to be updated in the next release as well